src/filefinder_rtp.cpp

```diff
@@ -178,6 +178,12 @@ void FileFinder_RTP::AddPath(StringView p) {
 	using namespace FileFinder;
 	auto fs = FileFinder::Root().Create(FileFinder::MakeCanonical(p));
 	if (fs) {
+		auto files = fs.ListDirectory();
+		if (files->size() == 0) {
```

Comment:
This change alone likely fixes the issue I had... 😉

Reply:
Yes. The rest is just for hypothetical other things or such a small improvement that you need tools to see. 😅